numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -280,13 +280,23 @@ addUfuncs(PyObject *dictionary) {
                     "inner on the last dimension and broadcast on the rest \n"
                     "     \"(i),(i)->()\" \n",
                     0, inner1d_signature);
+    /*
+     * yes, this should not happen, but I (MHvK) just spent an hour looking at
```

Comment:
Not sure this comment is necessary - any `PyObject *` allocation can fail due to memory limits, so we should always check results like this.

Reply:
Well, I'll maybe leave this as a breadcrump for the future - or to remind myself of hours waster...